rocAL/rocAL/source/api/rocal_api_augmentation.cpp

```diff
@@ -66,6 +66,24 @@ THE SOFTWARE.
 #include "commons.h"
 #include "context.h"
 #include "rocal_api.h"
+#include "image_source_evaluator.h"
+
+void
+get_max_resize_width_and_height(std::vector<unsigned> &src_size, std::vector<unsigned> &dst_size, RocalResizeScalingMode mode,
+                                std::vector<unsigned> &out_size) {
+    float max_aspect_ratio = 3.0f;
+
+    if(mode == ROCAL_SCALING_MODE_STRETCH) {
+        out_size[0] = dst_size[0] ? dst_size[0] : src_size[0];   // For width dimension
```

Comment:
Can we change this to std::min or std::max?

Reply:
We cannot use std::max in this case, because if we have a dst value we must assign it to out_size
If we use std::max , the max of the two values would be assigned here